doc/articles/guides/profiling-applications.md

```diff
@@ -0,0 +1,102 @@
+# Profiling Uno Platform Applications
+
+## Profiling .NET 6 Android applications
+
+As of Preview 7, .NET 6 provides the ability to do CPU profiling through [`dotnet-trace`](https://docs.microsoft.com/en-us/dotnet/core/diagnostics/dotnet-trace) for android applications.

```

Comment:
This will open the link in the user preferred language.

```suggestion
As of Preview 7, .NET 6 provides the ability to do CPU profiling through [`dotnet-trace`](https://docs.microsoft.com/dotnet/core/diagnostics/dotnet-trace) for android applications.

```